CMakeLists.txt

```diff
@@ -575,13 +586,10 @@ target_link_libraries (ripple_boost
 if (NOT DEFINED OPENSSL_ROOT_DIR)
   if (DEFINED ENV{OPENSSL_ROOT})
     set (OPENSSL_ROOT_DIR $ENV{OPENSSL_ROOT})
-  elseif (APPLE)
-    find_program (homebrew brew)
-    if (homebrew)
-      execute_process (COMMAND ${homebrew} --prefix openssl
-        OUTPUT_VARIABLE OPENSSL_ROOT_DIR
-        OUTPUT_STRIP_TRAILING_WHITESPACE)
-    endif ()
+  elseif (HOMEBREW)
+    execute_process (COMMAND ${HOMEBREW} --prefix openssl
+      OUTPUT_VARIABLE OPENSSL_ROOT_DIR
+      OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
no - this *queries* `homebrew` for the path to its `openssl` install location. If you don't have an `openssl` installed via `homebrew`, this will be empty on return.

Reply:
Oh, good! Thanks for the clarification.